### PR TITLE
Fixes oversight resulting in ALLCAPS traitor codewords; expands corpus

### PR DESCRIPTION
**Pull request by harmonyn**
See title, it really explains it all. The bloated line count is due to some partial alphabetization I started then decided I cba to finish

### CODE REVIEW

File: strings/ion_laws.json [hunk 1]
```diff
@@ -91,12 +91,14 @@
         "HOMOSEXUAL",
         "HETEROSEXUAL",
         "SEXUAL",
+	"BLOODY",
```
Comment (coiax) [inline strings/ion_laws.json]: inconsistent indentation

Comment (bgobandit) [inline strings/ion_laws.json]: fug

**Comment by ExcessiveUseOfCobblestone** [comment]
If you actually allcaps your codewords you are a dumbo to begin with.

File: strings/ion_laws.json [hunk 2]
```diff
@@ -353,6 +359,7 @@
         "STUN BATONS",
         "SUITS",
         "SUNGLASSES",
+	"SUPERMATTER SHARDS",
```
Comment (lordpidey) [inline strings/ion_laws.json]: Switch this one to just supermatter

File: strings/ion_laws.json [hunk 3]
```diff
@@ -471,7 +479,8 @@
         "BRING ME TO LIFE",
         "BRING ME THE GIRL",
         "SERVANTS",
-        "GREENTEXT"
+        "GREENTEXT",
```
Comment (lordpidey) [inline strings/ion_laws.json]: Let's not have OOC terms in this list please.

**Comment by bgobandit** [comment]
no, like, it pulls from the ion storm list, which is always UPPERCASE. but I forgot that the locations list failsafes to drinks. which results in all caps

File: strings/ion_laws.json (hunk 3 above)
Comment (bgobandit) [inline strings/ion_laws.json]: it's not OOC, there is an item called "greentext"

File: strings/ion_laws.json (hunk 2 above)
Comment (bgobandit) [inline strings/ion_laws.json]: the ion law this list is used for requires plurals

Comment (RemieRichards) [inline strings/ion_laws.json]: supermatter is plural and singular, it's like "sheep"
"get me the supermatter"
"pass me some supermatter"
"oh god, supermatter!"

**Comment by jughu** [comment]
Does anyone have SUPERMATTER SHARDS in their POCKETS!?

File: strings/ion_laws.json (hunk 2 above)
Comment (bgobandit) [inline strings/ion_laws.json]: no I mean like, the generated ion laws will be like "THERE ARE THREE BAWDY SUPERMATTER ON THE STATION" so it doesn't work in that sense

File: strings/ion_laws.json [hunk 4]
```diff
@@ -744,30 +760,32 @@
...
-        "CULTISTS",
-        "CENTCOM OFFICERS",
         "SPACE PIRATES",
         "TRAITORS",
         "MONKEYS",
-        "BEES",
-        "CARP",
-        "CRABS",
         "EELS",
-        "BANDITS",
```
Comment (Cheridan) [inline strings/ion_laws.json]: bias detected whoop whoop

Comment (bgobandit) [inline strings/ion_laws.json]: it was there before mr. cherry dan

Comment (Cheridan) [inline strings/ion_laws.json]: WHOOP WHOOP

File: strings/ion_laws.json [hunk 5]
```diff
@@ -744,30 +760,32 @@
...
         "LIZARDS",
         "PETES",
-        "BOMBS",
+	"RATVARIANS",
+        "XENOS",
         "FETISHES",
         "MEMES",
         "WIZARDS",
         "SYNDICATE AGENTS",
         "AHHHPERATIVES",
```
Comment (Militaires) [inline strings/ion_laws.json]: wrong

AWWWPERATIVES

Comment (coiax) [inline strings/ion_laws.json]: also there's some indentation issues here

**Review by lzimann — CHANGES_REQUESTED** [review]
fix the identation on the json(i can even do it for you if you don't want to touch every line manually)

File: strings/ion_laws.json (hunk 5 above)
Comment (bgobandit) [inline strings/ion_laws.json]: it's ahhhh

short o rhymes with cop, not with law

**Comment by bgobandit** [comment]
@lzimann I've tried fixing it but idk what's fucking up

**Comment by optimumtact** [comment]
You're using tabs when all the JSON files are formatted with spaces as per the JSON spec.

**Comment by optimumtact** [comment]
swampbandit please

**Review by lzimann — APPROVED** [review]

**Pull request merged**